portfolio/dash/overview_tab.py

```
@@ -0,0 +1,716 @@
+        prevent_initial_call=True,
+    )
+    def refresh_fdd_rules(_load, _refresh, site_id, hours, tab):
+        if tab and tab != "dashboard":
+            return no_update
+        if not site_id:
+            return html.P("Select a building.", style={"color": theme["muted"]})
+        fdd_data: dict = {"rules": [], "warnings": ["Central API not running — FDD rules need API + Edge."]}
+        try:
+            fdd_data = _api_get(f"/api/central/fdd-analytics/{site_id}?hours={hours or 24}")
+        except Exception:
+            pass
```

Ruff (0.15.15)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)